packages/client/README.md

```diff
@@ -38,8 +38,7 @@ for (const file of files) {
 ```
 
 ## Testing
-
-Run `npm test` to test the ESM code, CJS, and in the browser via `playwright-test`. 100% test coverage is required by the `hundreds` module.
+To test the client, you'll need two terminal windows open. In the first, start up the mock API by running `npm run mock:api`. In the second, you can then run `npm test` to test the ESM code, CJS, and in the browser via `playwright-test`. 100% test coverage is required by the `hundreds` module.
```

Comment:
So, `npm test` will sart the mock API for you, but if you run the tests individually like `npm run test:esm` then you'll need to manually start the mock API.